azkaban-common/src/main/java/azkaban/executor/container/DummyEventListener.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package azkaban.executor.container;
+
+import azkaban.event.EventListener;
+
+/**
+ * This class is a no-op implementation of {@link EventListener}
+ */
+public class DummyEventListener implements EventListener {
+
+  @Override
+  public void handleEvent(Object event) {
+  }
```

Comment:
Please log here that doing nothing kind of message.

Reply:
I appreciate the logging idea for debug purposes, but would this not be a bit excessive and not terribly useful in web server logs? Could we maybe log with DEBUG or TRACE level instead of INFO?

Reply:
Good idea, used log.trace